CHANGELOG.md

```diff
@@ -1,3 +1,115 @@
+# 1.0.0 (2023-09-25)
+
+This release represents a milestone in the development of the extension.
+The package is reorganize to be a collection of more self-contained
+mini-packages. Each with its own set of tests.
```

Comment:
```suggestion
mini-packages, each with its own set of tests.
```

either combine those two sentences, or give the second sentence an (auxiliary) verb